pkgs/applications/misc/plater/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, cmake
+, fetchFromGitHub
+, lib
+, libGLU
+, qt5
+}: stdenv.mkDerivation rec {
+  pname = "plater";
+  version = "2020-07-30";
+
+  src = fetchFromGitHub {
+    owner = "Rhoban";
+    repo = "Plater";
+    rev = "f8de6d038f95a9edebfcfe142c8e9783697d5b47";
+    sha256 = "0r20mbzd16zv1aiadjqdy7z6sp09rr6lgfxhvir4ll3cpakkynr4";
+  };
+
+  nativeBuildInputs = [ cmake qt5.wrapQtAppsHook ];
```

Use `mkDerivation` from libsForQt5

```suggestion
  nativeBuildInputs = [ cmake ];
```